Tests/XcodeGenKitTests/SourceGeneratorTests.swift

```diff
@@ -738,18 +770,24 @@ extension PBXProj {
     }
 
     private func getFileReference(group: PBXGroup, paths: [String], names: [String]) -> PBXFileReference? {
-
-        guard !paths.isEmpty else { return nil }
+        guard !paths.isEmpty else {
+            return nil
+        }
+        
         let path = paths.first!
         let name = names.first!
         let restOfPath = Array(paths.dropFirst())
         let restOfName = Array(names.dropFirst())
         if restOfPath.isEmpty {
             let fileReferences: [PBXFileReference] = group.children.compactMap { $0 as? PBXFileReference }
-            return fileReferences.first { $0.path == path && $0.nameOrPath == name }
+            fileReferences.forEach { print("path: \($0.path ?? "nil"), name: \($0.name ?? "nil")") }
+            return fileReferences.first { ($0.path == nil || $0.path == path) && $0.nameOrPath == name }
         } else {
             let groups = group.children.compactMap { $0 as? PBXGroup }
-            guard let group = groups.first(where: { $0.path == path && $0.nameOrPath == name }) else { return nil }
+            groups.forEach { print("path: \($0.path ?? "nil"), name: \($0.name ?? "nil")") }
```

Comment:
Can we remove these debugging print statements?

Reply:
Ah oops yes